paimon-core/src/main/java/org/apache/paimon/jdbc/JdbcCatalog.java

```diff
@@ -155,7 +156,8 @@ public List<String> listDatabases() {
                         row -> row.getString(JdbcUtils.DATABASE_NAME),
                         JdbcUtils.LIST_ALL_PROPERTY_DATABASES_SQL,
                         catalogKey));
-        return databases;
+
+        return databases.stream().distinct().collect(Collectors.toList());
```

Comment:
I am curious what the root cause of this is.
Why does databases contain the duplicate - is it ever valid? 
Are there other implications to leaving this list with duplicates? 
Would it not be better to fix this when the duplicate was added?
Also it would be good to add a junit.

Reply:
+1 curious

Reply:
There are two tables in JDBC catalog, `paimon_database_properties` contains database properties, `paimon_tables`  contains table , schema, catalog relations.  The same database will present in these two tables if creating a database and tables because Paimon will automatically add some properties when creating schema.
@davidradl  do you have any suggestion where to place the test?

Reply:
Can you add a test case?

Reply:
sure

Reply:
@JingsongLi  @davidradl add a test, please help to review again.